test/capi/capiPaint.cpp

```diff
@@ -161,4 +161,33 @@ TEST_CASE("Paint Bounds", "[capiPaintBounds]")
     REQUIRE(h == 100);
 
     REQUIRE(tvg_paint_del(paint) == TVG_RESULT_SUCCESS);
-}
\ No newline at end of file
+}
+
+TEST_CASE("Paint Dupliction", "[capiPaint]")
+{
+    Tvg_Paint* paint = tvg_shape_new();
+    REQUIRE(paint);
+
+    Tvg_Paint* paint_copy = tvg_paint_duplicate(paint);
```

Comment:
Maybe we can test if shape properties was duplicated properly?

Reply:
sure. done :)